src/views/udns/list/components/UserDefinedNetworkCreateForm.tsx

```diff
@@ -69,6 +70,12 @@ const UserDefinedNetworkCreateForm: FC<UserDefinedNetworkCreateFormProps> = ({
           )}
           rules={{ required: true }}
         />
+
+        <FormGroupHelperText>
+          {t(
+            'Dual-stack clusters may set 2 subnets (one for each IP family), otherwise only 1 subnet is allowed.  The format should match standard CIDR notation (for example, "10.128.0.0/16").',
+          )}
+        </FormGroupHelperText>
```

Comment:
please extract this as it's used in 2 places, any text change here would be prone to mistakes like this

Reply:
solved